### ISSUE TITLE
Wi-Fi Network Access

### ISSUE THREAD

**Issue by jgeib** [ifo-institute/ifohack2023#6]
We need to set up a working Wi-Fi environment to access the VMs at ifo. 
When using the Guest network I wasn't able to connect to the the VM. Also, I believe that this already quite slow network is overloaded by the parallel use of the cloud environment by about 80 participants.

**Comment by gerwolf** [comment]
The WiFi issues have been resolved: check this out, we have a new WiFi network called `ifoHack2023` (PW will be provided on-site).

You can simply scan the QR code below:

![ifoHack_WiFi-QR_Code.png](https://user-images.githubusercontent.com/38222418/233724647-55c8c816-1da7-45df-84cf-4c416ec37064.png)

😎

Several ports are open now that we're blocked by `Guest` and `Guest-secure`. Please check the connections in LES and canteen for speed and stability as well as different ports such as GitHub, ideally also on a private device. If there are any issues still, please open a new issue here.
